spec/support/features/session_helper.rb

```diff
@@ -50,28 +49,13 @@ def sign_in_with_warden(user)
       visit dashboard_index_path
     end
 
-    def sign_in_and_2fa_user(user = create_user([SecondFactor.find_by_name('Email')]))
+    def sign_in_and_2fa_user(user = create_user('555-555-5556'))
```

Comment:
This can be simplified to `def sign_in_and_2fa_user(user = create(:user, :signed_up))`
and the `create_user` method below can be removed.